src/store/accountActions.js

```diff
@@ -132,12 +133,14 @@ export function importWallet(wallet, name, description) {
                         type: 'ACCOUNT/IMPORT_WALLET',
                         accountId: result,
                     });
-                    dispatch(loadAccountBalance(result));
-                    if (address(result))
+                    // We should load balance and add the new account to the canonical list.
+                    if (address(result)) {
+                        dispatch(addAddress(result, name, description));
```

Comment:
I changed it `loadAccountsList()` which seems like overkill... is there another method you would suggest for pushing the new account just to the list of controlled accounts?